administrator/components/com_media/package.json

```diff
@@ -1,4 +1,34 @@
 {
+	"name": "com_media",
+	"version": "3.0.0",
+	"description": "Component for managing site media",
+	"license": "GPL-3.0",
+	"author": "",
+	"scripts": {
+		"build": "npm run build-css & npm run build-js",
+		"build-css": "node-sass ./resources/styles/mediamanager.scss ../../../media/com_media/css/mediamanager.css",
+		"build-js": "browserify ./resources/scripts/mediamanager.js > ../../../media/com_media/js/mediamanager.js"
+	},
+	"dependencies": {
+		"vue": "^2.2.4",
+		"vuex": "^2.2.1",
+    "vue-focus": "^2.1.0",
+	},
+	"devDependencies": {
+		"babel-core": "^6.24.0",
+		"babel-preset-es2015": "^6.24.0",
+		"babelify": "^7.2.0",
+		"browserify": "^13.0.1",
+		"node-sass": "^4.5.1",
+		"vueify": "^9.4.1"
+	},
+	"browserify": {
+		"transform": [
+			"vueify",
+			"babelify"
+		]
+	}
+  
```

Comment:
and kindly run "npm run build" in the media manager root to add the updated mediamanager js file.